data/json/items/armor/robofac_armor.json

```diff
@@ -378,7 +378,7 @@
     "type": "ARMOR",
     "copy-from": "robofac_mantle",
     "name": "Hub 01 prototype mantle",
-    "description": "3D printed upper body armor, despite its rough finish it's still lightweight and it seems a lot of thought went into the ergonomics.  This is made of 3D printed plastic with an unhardened metal core to give some reinforcement.  Designed to clip to the chest and shoulders the interior side reads ATTACH TO MODULAR DEFENSE SYSTEM.",
+    "description": "3D printed armor for the upper body, made of thick and flexible polymer with metallic fibers for reinforcement.  Several buckles and straps allow it to be adjusted to fit you comfortably.  Despite its rough finish, it's clear that a lot of thought has gone into the ergonomics and weight.  Designed to clip to the chest and shoulders, its interior reads \"ATTACH TO MODULAR DEFENSE SYSTEM\".",
```

Comment:
Might be worth describing the metal as something like "soft" here to further differentiate materials.